src/main/java/com/google/sps/data/backgroundTaskManagerInterface.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+import java.util.Optional;
+import java.util.List;
+
+/** Interface for the backgroundTaskManager class. */
+public interface backgroundTaskManagerInterface { 
+/**
+   * Checks VM status to make sure they are healthy. 
+   */
+    void checkVM();
```

Comment:
Kinda vague. What does "make sure they are healthy mean"? How about:

void updateInstances();

All it does it make a call to list all the instances being used and then updates the VM status in our database. Talk to Taniece, there's a field in the Instance entity for VM status.